README.md

```diff
@@ -515,6 +516,80 @@ networks anything more precise than a millisecond will not generally be useful.
 See also [--rate &lt;requests/second&gt;](#--rate-requestssecond) below for
 rate specification of transactions.
 
+### HTTP/2 Transaction Abort
+
+In some cases, there might be a need to test the behavior of the proxy when the
+client aborted the request, or when the server aborted the response, either because
+of an unexpected error or intentionally cancelling the connection. The way transactions
+are aborted is done by sending a RST_STREAM frame along with an error code, the user
+can sepcify when the RST_STREAM frame is sent, and what error code it has.
+
+The following sample replay file demonstrate how to specify such a test scenario:
+
+```YAML
+sessions:
+- protocol:
+  - name: http
+    version: 2
+  - name: tls
+    sni: test_sni
+  - name: tcp
+  - name: ip
+    version: 4
+  transactions:
+  - client-request:
```

Comment:
Did you see my comment here:
https://github.com/yahoo/proxy-verifier/pull/221#pullrequestreview-1109388374

Rather than specify an "after", it would be good to specify frames like so:

```
sessions:
  transactions:
    -client-request:
       frames:
         HEADERS:
         RST_STREAM
```

Reply:
I thought that's for the next PR?